doc/releases/release-notes-4.2.rst

```diff
@@ -63,6 +63,12 @@ New APIs and options
   like you need to add more details, add them in the API documentation code
   instead.
 
+* Bluetooth
+
+  * Host
+
+    * :c:func:`bt_le_get_local_features`
```

Comment:
I know it is possible to click on the link, but it would still be nice to add a short description of what it does :)
```suggestion
    * :c:func:`bt_le_get_local_features` returns the feature set supported by the Bluetooth Controller.
```

Reply:
Normally I'd do that too, but the text right above says:

> Link to new APIs here, in a group if you think it's necessary, no need to get
  fancy just list the link, that should contain the documentation. If you feel
  like you need to add more details, add them in the API documentation code
  instead.

I read that as we should only provide links here and then the documentation of the added function should provide the rest